### DISCUSSION TITLE
Console error after upgrading JS modules to 2.4.0

### DISCUSSION THREAD

**Discussion by fcaballero** [perspective-dev/perspective#2335] [Q&A]
### Steps to Reproduce:
 I just updated the next package to 2.4.0 and the table was not rendering
@finos/perspective
@finos/perspective-viewer
@finos/perspective-viewer-d3fc
@finos/perspective-viewer-datagrid
@finos/perspective-workspace
@finos/perspective-webpack-plugin

### Expected Result:
No console error

### Actual Result:
Getting error after updating libraries
![Screenshot 2023-08-10 at 12 26 29](https://github.com/finos/perspective/assets/1280241/e460f899-ac4e-4b22-bdb1-326c2ac9a39f)

### Environment:
Version 2.4.0 of Perspective in Chrome

### Additional Context:
Besides just updating, I also removed all node modules and did an installation from scratch, the result was the same.

**Comment by texodus** [comment]
This is _not_ a repro. You _need_ to provide a _complete repro_ if you want us to help debug something, we simply do not have time to go try experimenting to reproduce your issue.

Perspective has a test suite that runs over 22k test cases across operating systems, browsers and runtime versions. 2.4.0 passed all of them.  The [GitHub Pages site](https://perspective.finos.org/) is on 2.4.0 and does not display problems, nor do any of the [dozens of example web examples](https://perspective.finos.org/examples/).
